tests/e2e/lib.sh

```diff
@@ -143,7 +143,8 @@ export_test_environment() {
     ci_export ROX_VULN_MGMT_WORKLOAD_CVES "${ROX_VULN_MGMT_WORKLOAD_CVES:-true}"
     ci_export ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL "${ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL:-true}"
     ci_export ROX_VULN_MGMT_NODE_PLATFORM_CVES "${ROX_VULN_MGMT_NODE_PLATFORM_CVES:-true}"
-    ci_export ROX_VULN_MGMT_2_MISC_IMPROVEMENTS "${ROX_VULN_MGMT_2_MISC_IMPROVEMENTS:-true}"
+    ci_export ROX_VULN_MGMT_2_GA "${ROX_VULN_MGMT_2_GA:-true}"
+    ci_export ROX_VULN_MGMT_ADVANCED_FILTERS "${ROX_VULN_MGMT_ADVANCED_FILTERS:-true}"
```

Comment:
There is another place recently added in this file. See step 4
https://github.com/stackrox/stackrox/tree/master/ui/apps/platform#add-a-feature-flag-to-frontend-code

Reply:
Oh interesting, didn't know about this one. Could you explain this a bit more?

Reply:
For tests that install via operator instead of helm chart.